src/main/java/games/strategy/engine/lobby/server/login/LobbyLoginValidator.java

```diff
@@ -14,12 +14,12 @@
 
 import games.strategy.engine.framework.startup.ui.InGameLobbyWatcher;
 import games.strategy.engine.lobby.server.LobbyServer;
-import games.strategy.engine.lobby.server.userDB.BadWordController;
-import games.strategy.engine.lobby.server.userDB.BannedMacController;
-import games.strategy.engine.lobby.server.userDB.BannedUsernameController;
+import games.strategy.engine.lobby.server.userdb.BadWordController;
+import games.strategy.engine.lobby.server.userdb.BannedMacController;
+import games.strategy.engine.lobby.server.userdb.BannedUsernameController;
 import games.strategy.engine.lobby.server.userDB.DBUser;
```

Comment:
Checkstyle: "Wrong lexicographical order for 'games.strategy.engine.lobby.server.userDB.DBUser' import. Should be 
 before 'games.strategy.engine.lobby.server.userdb.BannedUsernameController'."